src/style.scss

```diff
@@ -0,0 +1,65 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.form {
+  &__input {
+    box-sizing: border-box;
+    display: flex;
+    width: 100%;
+
+    margin-top: 20px;
+    padding: 0;
+    font-family: Avenir, Arial, sans-serif;
+    font-weight: 300;
+
+    border: 1px solid #e1e7ed;
+    border-radius: 4px;
+    box-shadow: 0 1px 8px #3d4e611a;
+
+    background-image: url(images/Search.svg);
+    background-repeat: no-repeat;
+
+    &--big {
+      height: 70px;
+      background-size: 19px;
+      background-position: 26px center;
+      padding-left: 62px;
+      font-size: 16px;
+      line-height: 22px;
+    }
+
+    &--small {
+      box-sizing: border-box;
```

Comment:
You've used `box-sizing: border-box;` in both `.form__input--big` and `.form__input--small`. Since they share this property, it's more efficient to set it in the parent `.form__input` class. This is a code quality issue.